orcestra_book/reports/HALO-20240906a.md

```diff
@@ -1,4 +1,4 @@
----
+4---
```

Comment:
this 4 should not be here.  I tried to fix it, but was denied permission to push the fix back via your branch.